lib/docs/src/checkbox.stories.tsx

```diff
@@ -1,27 +1,115 @@
+import { useState } from "react";
+import { DivPx, Checkbox } from "../../core/src";
+import { Utils } from "./utils";
 import { Meta } from "@storybook/react";
-import { Checkbox, DivPx } from "../../core/src";
 
-export default {
-	title: "Draft/Checkbox",
+const meta: Meta = {
+	title: "Components/Checkbox",
 	component: Checkbox,
-} as Meta;
-
-export const Primary = (): JSX.Element => (
-	<div>
-		<Checkbox indeterminate>Checkbox</Checkbox>
-		<DivPx size={8} />
-		<Checkbox defaultChecked>Checkbox</Checkbox>
-		<DivPx size={8} />
-		<Checkbox>Checkbox</Checkbox>
-		<DivPx size={8} />
-		<Checkbox disabled indeterminate>
-			Checkbox
-		</Checkbox>
-		<DivPx size={8} />
-		<Checkbox disabled defaultChecked>
-			Checkbox
-		</Checkbox>
-		<DivPx size={8} />
-		<Checkbox disabled>Checkbox</Checkbox>
-	</div>
-);
+	argTypes: {
+		checked: Utils.arg("boolean", "Visual"),
+		indeterminate: Utils.arg("boolean", "Visual"),
+		disabled: Utils.arg("boolean", "Visual"),
+		children: Utils.arg(null, "Controlled"),
+		setChecked: Utils.arg(null, "Controlled"),
+		defaultChecked: Utils.arg(null, "Uncontrolled"),
+		forwardedRef: Utils.arg(null, "Uncontrolled"),
+	},
+};
+
+Utils.page.component(meta, { sticky: true, shots: [] });
+
+export default meta;
+
+interface Props {
+	indeterminate: boolean;
+	disabled: boolean;
+	checked: boolean;
+}
+
+export const Primary = (props: Props): JSX.Element => {
+	return (
+		<>
+			<Checkbox
+				defaultChecked
+				checked={props.checked}
```

Comment:
"defaultChecked" and "checked" should not be used at the same time since one is for uncontrolled mode and one is for controlled mode (in fact there might be a warning from React if you open the console)

What should be done here is to disable the "defaultChecked", and ensure "checked" is always defined:

```ts
checked={props.checked ?? true}
```